vuepress/docs/next/tutorials/ecr/deploy-components-without-bundle.md

```diff
@@ -1,46 +1,54 @@
+In this tutorial you will learn to apply an Entando Blueprint to a plugin and Micro Frontend generated with the Entando Component Generator in a running cluster, without the need for a bundle.
 
 ## Prerequisites
 
--   An Entando Plugin built with the Entando Component Generator and populated with micro frontends
+-   An Entando Plugin built with the Entando Component Generator and populated with Micro 
+Frontends. [Generate a plugin and Micro Frontend here](../backend-developers/generate-microservices-and-micro-frontends.md). 
+
+-   Node and NPM installed on your machine (use LTS version)
 
--   Node and NPM are installed on your machine (use LTS version)
+-   Docker installed on your machine and able to upload images to Docker Hub or an image repository of your choice
 
--   Docker is installed on your machine and you are able to upload images to docker-hub or an image repository of your choice
+-   An instance of the Entando platform running Kubernetes. See [Getting Started](../../docs/getting-started/README.md)
 
--   An installed instance of the Entando platform running Kubernetes. See [Getting Started](../../docs/getting-started/README.md)
+::: tip Note
+[Here is a definition of an Entando plugin,](../../docs/ecr/ecr-bundle-details.html#plugin) including information on the runtime contract required for using non-Java based plugins.
+:::
 
-## Steps
 
-## 1. Generate a docker image for your microservice
+## Generate a Docker image 
 
 JHipster uses the JIB Maven plugin to generate a docker image for your
 microservice.
 
-The name of the output image generated with JIB will be composed by:
-- The organization you chose during the setup wizard (by default that's set to `entando`)
+The name of the output image generated with JIB during the setup wizard is defined with:
+- The organization name you chose, default is set to `entando`
 - The name of the application
 - Version `0.0.1-SNAPSHOT`
 
-You can build the docker image with this command
+For example, a custom organization name of `myorg` and the application name of `jhipster` will result in a docker image named `myorg/jhipster:0.0.1-SNAPSHOT`.
 
+1. Build the docker image:
+    ```
     ./mvnw -Pprod clean package jib:dockerBuild
+    ```
+
+
 
-If for example during setup wizard you chose a custom organization `myorg` and the set the application name to `jhipster` the resulting docker image is going to be `myorg/jhipster:0.0.1-SNAPSHOT`
+The output image name can also be set by customizing the `./mvnw` command using the `-Djib.to.image` parameter. With the organization name `myneworg`, name `myapp` and version `latest` use:
+```
+ ./mvnw -Pprod clean package jib:dockerBuild -Djib.to.image=myneworg/myapp:latest
+```
 
-> **Note**
->
-> The output image name can be changed in the `pom.xml` file by configuring 
-> the `plugins.plugin.jib-maven-plugin.configuration.to.image` tag
+**Note**
```

Comment:
Is this something that needs to be done? If so it should be an instruction, right?

Reply:
Fine as-is